model/README.md

```diff
@@ -0,0 +1,20 @@
+# Libero Model
+
+## Archimate 
+
+> Archi® is a free, open source, cross-platform tool and editor to create ArchiMate models.
```

Comment:
Discovered this is a [separate language from UML](https://en.wikipedia.org/wiki/ArchiMate):

> UML and BPMN are meant for a specific use and they are quite heavy – containing about 150 (UML) and 250 (BPMN) modeling concepts whereas ArchiMate works with just about 50 (in version 2.0). The goal of ArchiMate is to be ”as small as possible”, not to cover every edge scenario imaginable.